demos/astro-playground/astro.config.mjs

```diff
@@ -27,6 +27,9 @@ export default defineConfig({
     react({ include: ['**/react/*'] }),
     sentry({
       debug: true,
+      sourceMapsUploadOptions: {
+        enabled: false, // tmp deactivate until version after 7.83.0
+      },
```

Comment:
fixed by https://github.com/getsentry/sentry-javascript/pull/9688 but we need to bump the SDK here, too. 

Also reminder to adjust the minimum required SDK version before the launch